scripts/translations/base_string_script.py

```diff
@@ -10,8 +10,8 @@
 
 class BaseStringScript:
 
-  def ProcessTag(self, lines):
-    return lines
+  def ProcessTag(self, line):
```

Comment:
@samtstern oh, got it. Yeah, I was confused when writing my script because I thought it was all lines of the whole file. I see what you meant now—either way, #namingishard 😂